doc/ref/configuration/master.rst

```diff
@@ -1941,6 +1941,42 @@ the cloud profile or master config file, no templating will be performed.
 
     userdata_template: jinja
 
+.. conf_master:: jinja_line_statement_prefix
+
+``jinja_line_statement_prefix``
+---------------------
+
+.. versionadded:: Oxygen
+
+Default: ``''``
+
+If this is set to a string that is not empty, any line starting with this prefix
+will be interpreted as a line statement by the jinja renderer.
+Defaults to ``''`` and corresponds
+to the Jinja environment init variable ``line_statement_prefix``.
+
+.. code-block:: yaml
+
+    jinja_line_statement_prefix: ''
+
+.. conf_master:: jinja_line_comment_prefix
+
+``jinja_line_comment_prefix``
+---------------------
```

Comment:
Please extend the dashed line to match the length of the line above. Using different lengths won't make docs builds fail, but it will muddy the output with a warning.